spk/kavita/src/service-setup.sh

```diff
@@ -0,0 +1,32 @@
+
+# Kavita service setup
+KAVITA="${SYNOPKG_PKGDEST}/share/Kavita"
+
+# Kavita uses custom Config and PID directories
+HOME_DIR="${SYNOPKG_PKGDEST}/share"
+CONFIG_DIR="${SYNOPKG_PKGVAR}/config"
+PID_FILE="${SYNOPKG_PKGVAR}/kavita.pid"
+
+SVC_BACKGROUND=y
+SVC_WAIT_TIMEOUT=90
+
+service_prestart ()
+{
+    # Replace generic service startup, fork process in background
+    echo "Starting Kavita at ${HOME_DIR}" >> ${LOG_FILE}
+    cd ${HOME_DIR} || exit 1;
+    ${KAVITA} >> ${LOG_FILE} 2>&1 &
+    echo "$!" > "${PID_FILE}"
+}
```

Comment:
IMO this can also be achieved with
```
SVC_CWD=${HOME_DIR}
SVC_WRITE_PID=y
SERVICE_COMMAND="${KAVITA} >> ${LOG_FILE}"
```

this requires also `SVC_BACKGROUND=y` that is already defined above.

Reply:
I'll try that and see if it works.

Reply:
It works! The package can now look a lot more like the others. Thanks for that.

Reply:
I'm guessing that it can be simplified even further by removing the `>> ${LOG_FILE}` since this would be a default as well?

Reply:
Yup, logging still works without this.